src/main/java/edu/umich/tl/ApiCallHandler.java

```diff
@@ -0,0 +1,82 @@
+package edu.umich.tl;
+
+import java.io.IOException;
+import java.util.ArrayList;
+
+import org.apache.commons.logging.Log;
+import org.apache.commons.logging.LogFactory;
+import org.apache.http.HttpResponse;
+import org.apache.http.NameValuePair;
+import org.apache.http.client.HttpClient;
+import org.apache.http.client.methods.HttpGet;
+import org.apache.http.client.methods.HttpUriRequest;
+import org.apache.http.impl.client.DefaultHttpClient;
+import org.apache.http.message.BasicNameValuePair;
+
+public class ApiCallHandler {
```

Comment:
The class name is too generic for my tastes - This class will call Canvas either directly or indirectly, so I'd call it canvasAPIcallHandler to be more clear